proto/swisstronik/compliance/genesis.proto

```diff
@@ -10,15 +10,16 @@ option go_package = "swisstronik/x/compliance/types";
 // GenesisState defines the compliance module's genesis state.
 message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
-  repeated IssuerGenesisAccount issuers = 2;
+  repeated GenesisIssuerDetails issuerDetails = 2;
```

Comment:
Looks like we should prepare some upgrade to be able start node with existing genesis.json

Reply:
It doesn't break importing genesis when we migrate to new version.
If there's a migration function for specific module, module manager (module.Mananger) won't call `InitGenesis` within [`RunMigrations`](https://github.com/SigmaGmbH/swisstronik-chain/pull/115/files#diff-1999b5d151142c0e25aa3971448f893be3bcaeca2dd4404fea9ab8d7d4f001a1R16).